Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -224,6 +224,10 @@ Permissions should be explicitly requested for every needed permission, even if
 
 For example if both `READ_EXTERNAL_STORAGE` and `WRITE_EXTERNAL_STORAGE` are listed in the app manifest but only permissions are granted for `READ_EXTERNAL_STORAGE`, then requesting `WRITE_LOCAL_STORAGE` will automatically have permissions without user interaction because they are in the same group and not explicitly requested.
 
+#### Permisson Analysis
+
+Always check whether the application is requesting sensible permissions. Make sure that no permissions are requested which are not related to the goal of the app. For instance: a single-player game that requires access to `android.permission.WRITE_SMS`, might not be a good idea.
```

Comment:
I think we should stick to the android documentation wording and call it dangerous permissions instead of sensible and make a link to a list of the dangerous permissions:  https://developer.android.com/guide/topics/permissions/overview#permission-groups

Reply:
I meant to say: for any given permission: it is bes tot make sure that you checking whether asking these permissions make sense. So not just the `dangerous` ones.

Reply:
After discussion oflfine, changed it to `Always check whether the application is requesting permissions it actually needs`